static/_redirects

```diff
@@ -496,8 +495,12 @@
 /docs/admin/authorization/webhook/     /docs/reference/access-authn-authz/webhook/ 301
 /docs/admin/authorization/     /docs/reference/access-authn-authz/authorization/ 301
 /docs/admin/high-availability/building/     /docs/setup/production-environment/tools/kubeadm/high-availability/   301
+
 /code-of-conduct/     /community/code-of-conduct/   301
 /values/     /community/values/   302
+/dockershim /blog/2020/12/02/dockershim-faq/ 302
+/dockershim/ /blog/2020/12/02/dockershim-faq/ 302
```

Comment:
I guess that just one rule with trailing slash is enough, because that one with trailing slash can handle for both cases. 😊

Proof: for `/values/`, there is only one rule with trailing slash,
and both https://kubernetes.io/values and https://kubernetes.io/values/ redirects to https://kubernetes.io/community/values/.

```suggestion
/dockershim/ /blog/2020/12/02/dockershim-faq/ 302
```